lib/src/component_declaration/built_redux_component.dart

```diff
@@ -51,33 +103,34 @@ abstract class BuiltReduxUiProps<V extends Built<V, B>, B extends Builder<V, B>,
 /// __Note:__ [Substate] must be a comparable object to avoid unnecessary redraws,
 /// it is recommended that [Substate] either be a primitive, built_value, build_collection or an Object
 /// that overrides `==`.
-///
-/// __Warning:__ This will be deprecated in an upcoming release in favor of a different approach to
-/// creating a `built_redux` component.
-abstract class BuiltReduxUiComponent<
+abstract class BuiltReduxUiComponent2<
```

Comment:
Honestly I don't think we should even have a V2 of this since we'll already be deprecating `BuiltReduxUiComponent` as part of the `connect` work, but I'm fine merging it and removing it in a follow-up